src/utils/flag.js

```diff
@@ -55,4 +55,10 @@ Figure.defaultProps = Object.assign({}, Figure.defaultProps, {
   blurred: flag('amount_blur') ? true : false
 })
 
+flag.store.on('change', function(flagName) {
+  if (flagName == 'amount_blur') {
+    Figure.defaultProps.blurred = flag('amount_blur')
```

Comment:
Does it reload also the mounted Figure component?

Reply:
No, at this point in the app, you are in the settings, there are no mounted Figure components.

Reply:
OK thanks 👍

Reply:
Another solution would be in banks to use an "overloaded" FlagFigureComponent component that would observe flags through `connect`. This component would also need to be used in FigureBlock. That'd mean FigureBlock should have a FigureComponent prop set by default to FlagFigureComponent.

Reply:
I've moved the discussion there as I think the fix is OK and that the discussion could take some time to resolve : https://github.com/cozy/cozy-ui/issues/1478